core/components/molecules/list/list.story.js

```diff
@@ -1,51 +1,287 @@
+            temporellentesque. Risus ultricies tristique nulla aliquet enim. Proin libero nunc
+            consequat interdum varius sit amet. Scelerisque viverra mauris in aliquam sem fringilla
+            ut morbi tincidunt. Tincidunt arcu nonLorem ipsum dolor sit amet, consectetur adipiscing
+            elit, sed do eiusmod temporellentesque. Risus ultricies tristique nulla aliquet enim.
+            Proin libero nunc consequat interdum varius sit amet. Scelerisque viverra mauris in
+            aliquam sem fringilla ut morbi tincidunt. Tincidunt arcu nonLorem ipsum dolor sit amet,
+            consectetur adipiscing elit, sed do eiusmod temporellentesque. Risus ultricies tristique
+            nulla aliquet enim. Proin libero nunc consequat interdum varius sit amet. Scelerisque
+            viverra mauris in aliquam sem fringilla ut morbi tincidunt. Tincidunt arcu non
+          </List.Drawer>
+        </List.Item>
+      </List>
+    )
+  }
+}
+
+storiesOf('List', module).add('Default structure', () => (
+  <Example title="Default structure">
+    <ExampleList />
+  </Example>
+))
+
+storiesOf('List', module).add('Simple list', () => (
+  <Example title="Simple list">
+    <List>
+      <List.Item>Item 1</List.Item>
+      <List.Item>Item 2</List.Item>
+      <List.Item>Item 3</List.Item>
     </List>
   </Example>
 ))
 
-storiesOf('List', module).add('with stack', () => (
-  <Example title="with stack">
-    <List label="Social">
-      <Stack>
-        <div>github</div>
-        <div>GitHub</div>
-        <Switch on />
-      </Stack>
-      <Stack>
-        <div>google-oauth2</div>
-        <div>Google</div>
-        <Switch />
-      </Stack>
+storiesOf('List', module).add('with StackLayout', () => (
+  <Example title="with StackLayout">
+    <List>
+      <List.Item>
+        <StackLayout distribution="spaceBetween">
+          <Heading size={4}>Heading</Heading>
+          <ButtonGroup compressed>
+            <Button icon="pencil" />
+            <Button icon="copy" />
+            <Button icon="delete" />
+          </ButtonGroup>
+        </StackLayout>
+      </List.Item>
+
+      <List.Item>
+        <StackLayout distribution="spaceBetween">
+          <Heading size={4}>Heading</Heading>
+          <ButtonGroup compressed>
+            <Button icon="pencil" />
+            <Button icon="copy" />
+            <Button icon="delete" />
+          </ButtonGroup>
+        </StackLayout>
+      </List.Item>
+
+      <List.Item>
+        <StackLayout distribution="spaceBetween">
+          <Heading size={4}>Heading</Heading>
+          <ButtonGroup compressed>
+            <Button icon="pencil" />
+            <Button icon="copy" />
+            <Button icon="delete" />
+          </ButtonGroup>
+        </StackLayout>
+      </List.Item>
+
+      <List.Item>
+        <StackLayout distribution="spaceBetween">
+          <Heading size={4}>Heading</Heading>
+          <ButtonGroup compressed>
```

Comment:
No need for `compressed` here as the usual pattern for actions in lists uses buttons with spacing between them.